core/src/main/java/org/jdbi/v3/core/mapper/reflect/FieldMapper.java

```diff
@@ -146,10 +146,10 @@ public T map(ResultSet rs, StatementContext ctx) throws SQLException {
 
                     findColumnIndex(paramName, columnNames, columnNameMatchers, () -> debugName(field))
                         .ifPresent(index -> {
-                            QualifiedType type = QualifiedType.of(field.getGenericType())
+                            QualifiedType<?> type = QualifiedType.of(field.getGenericType())
                                 .with(getQualifiers(field));
                             ColumnMapper<?> mapper = ctx.findColumnMapperFor(type)
-                                .orElse((r, n, c) -> r.getObject(n));
+                                .orElse((ColumnMapper) (r, n, c) -> r.getObject(n));
```

Comment:
Probably wants `@SuppressWarnings("raw")` here too

Reply:
Ditto, not seeing this as resolved.